nixos/modules/programs/trippy.nix

```diff
@@ -0,0 +1,24 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.programs.trippy;
+in
+
+{
+  options = {
+    programs.trippy = {
+      enable = lib.mkEnableOption (lib.mdDoc "trippy");
+    };
+  };
+
+  config = lib.mkIf cfg.enable {
```

Comment:
Also add it to `environment.systemPackages`?

Reply:
We shouldn't need it since there is already a wrapper